src/test/java/org/tugraz/sysds/test/functions/recompile/CSVReadInFunctionTest.java

```diff
@@ -82,7 +82,9 @@ private void runCSVReadInFunctionTest(String testname, boolean withMtD) {
 			
 			String HOME = SCRIPT_DIR + TEST_DIR;
 			fullDMLScriptName = HOME + testname + ".dml";
-			programArgs = new String[]{"-stats", "-explain",
+			programArgs = new String[]{
```

Comment:
The following two tests are failing in this file:

1) testCSVReadFunctionMTD

2) testCSVReadNoFunctionMTD